ci-operator/jobs/openshift/cincinnati/openshift-cincinnati-gh-pages-postsubmits.yml

```diff
@@ -0,0 +1,33 @@
+postsubmits:
+  openshift/cincinnati:
+  - agent: kubernetes
+    branches:
+    - gh-pages
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
```

Comment:
I think you have to remove this label, which means you can remove the whole `labels` dictionary

/cc @openshift/openshift-team-developer-productivity-test-platform 

```suggestion
```

Reply:
Yeah, if this job is not generated by Prowgen, it should not have this label